Code/Tools/FBuild/Documentation/docs/functions/print.html

```diff
@@ -29,9 +29,26 @@ <h1>Print</h1>
 <div id='print' class='newsitemheader'>Summary</div>
 <div class='newsitembody'>
   <p>
-  Prints some text during BFF parsing.
+  Prints some text or variable state during BFF parsing.
   </p>
-  <div class='code'>Print( "String or $VarName$" )</div>
+  <div class='code'>Print( "String with optional $VarName$" ) // (1)
+Print( .VarName )                         // (2)
+Print( /String with optional $VarName$/opts )        // (3)</div>
+<ol>
+  <li>Prints a string, embedding any scalar variables included.  To print a $, escape it with a ^.  
+      Arrays and structs cannot be printed this way.</li>
+  <li>Prints a variable.  If the variable is an array or struct, it will recursively print the members.</li>
```

Comment:
I didn't know about this variation for a long time; I may have only found out about it from looking at the code.